classes/exercises.py

```diff
@@ -26,3 +26,39 @@
 #    because we already know that all of them should be 60.
 
 # 2.1. Create a `Equilateral` class method that receives a side length and calculates the triangle area.
+
+class Triangle(object):
+
+    number_of_sides = 3
+
+    def __init__(self, angle1, angle2,angle3):
+        self.angle1 = angle1
+        self.angle2 = angle2
+        self.angle3 = angle3
+
+    def check_angles(self):
+        if sum([self.angle1,self.angle2,self.angle3]) == 180:
+            return True
+        return False 
+            
+
+    def get_side_lengths(self, perimeter):
+        def get_K(self,perimeter):
+            sum_sin = math.sin(math.radians(self.angle1)) + math.sin(math.radians(self.angle2)) + math.sin(math.radians
+            (self.angle3))
+            return perimeter/sum_sin
+            
+        k = get_K(self,perimeter)
+        list = [k*math.sin(math.radians(self.angle1)),k*math.sin(math.radians(self.angle2)),k*math.sin(math.radians(self.angle3))]
+
+        return [round(item, 2) for item in list]
+
+class Equilateral(Triangle):
+      def __init__(self):
+        self.angle1 = 60
+        self.angle2 = 60
+        self.angle3 = 60

```

Comment:
you should use the parent constructor to initialize this class
```suggestion
        super().__init__(60,60,60)
```